modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/dcrm/api/OAuthDCRMTestCase.java

```diff
@@ -314,4 +315,76 @@ private String getAuthzHeader() {
         return "Basic " + Base64.encodeBase64String((username + ":" + password).getBytes()).trim();
     }
 
+    private JSONObject getPayload(HttpResponse response) throws IOException {
+
+        BufferedReader rd = new BufferedReader(new InputStreamReader(response.getEntity().getContent()));
+        Object responseObj = JSONValue.parse(rd);
+        EntityUtils.consume(response.getEntity());
+        return (JSONObject) responseObj;
+    }
+
+    @Test(alwaysRun = true, groups = "wso2.is", priority = 9, description = "Create a service provider with " +
+            "additional OIDC properties")
+    public void testCreateServiceProviderRequestWithAdditionalParameters() throws IOException {
```

Comment:
Shall we add a test for updates as well?

Reply:
added